src/python/T0Component/Tier0Feeder/MultipleAgents/HelperAgent.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from T0Component.Tier0Feeder.MultipleAgents.BaseAgent import BaseAgent
+
+class HelperAgent(BaseAgent):
+    """
+    Helper agent. The streams processed by this one are explicitely defined in the tier0Config. 
+    helperRole has to match
+
+    Use it with
+    config.Tier0Feeder.AgentRole = "SecondAgent" (or "ThirdAgent" ..., follow the configuration file)
+    Note: if the helperName is none or unknown, returned values will be empty and no data is injected
+    """
+    def __init__(self, tier0Config, helperName = None):
+        
+        BaseAgent.__init__(self, tier0Config)
+        self.name = helperName
+        if self.name and self.name in self.helperAgentStreams:
+            self.streams = self.helperAgentStreams[self.name]
+        else:
+            self.streams = []
```

Comment:
If there is no match between the given name and the keys in the dictionary defined in the configuration, no streams will be processed by the helper agent.